src/content/docs/tutorial-app-performance/guide-to-monitoring-core-web-vitals.mdx

```diff
@@ -161,20 +160,26 @@ If you want to investigate something particular, like how a backend change or pe
   **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Monitor > Web vitals > (add filters)**: Use the filter tool to filter core web vitals by key attributes.
 </figcaption>
 
-### Filtering examples
-
 Though you can filter by nearly any key attribute, read on for some useful examples.
 
-#### Filter by device type
+<CollapserGroup>
+    <Collapser
+        id="device"
+        title="By device type"
+    >
 
 Let's say you want to compare desktop vs. mobile performance and how core web vitals differ between the two platforms. To investigate this in **Web vitals**:
 
 1. In the **Filter by** field, set **Attribute** to`deviceType` and **Value** to `desktop`.
 2. In a duplicate browser, set **Attribute** to`deviceType` and **Value** to `mobile`, then compare the results. 
 
-#### Filter by user information
+    </Collapser>
+    <Collapser
+        id="user-info"
+        title="By user information"
+    >
 
-Let's say an end-user complained about performance issues on your site’s checkout page on November 1, around 12pm, and you want to see what happened for this user. To investigate this in **Web vitals**:
+Let's say an end-user reported performance issues on your site’s checkout page on November 1, around 12pm, and you want to see what happened for this user. To investigate this in **Web vitals**:
```

Comment:
"To investigate this in" sounds a little odd. How about "You can investigate this in" instead?